helm/falco-app/charts/falco/values.yaml

```diff
@@ -290,7 +290,7 @@ falco:
   # You can configure the address to bind and expose it.
   # By modifying the threadiness configuration you can fine-tune the number of threads (and context) it will use.
   grpc:
-    enabled: false
+    enabled: true
```

Comment:
Can you define this here instead?
https://github.com/giantswarm/falco-app/blob/master/helm/falco-app/values.yaml

That way we have an easier time later on updating the subchart here.